lisk/package.json

```diff
@@ -74,18 +74,11 @@
 		"rewire": "4.0.1",
 		"sinon": "7.2.2",
 		"sinon-chai": "3.3.0",
-		"snyk": "^1.122.0",
 		"stampit": "4.2.0",
 		"supertest": "3.3.0"
 	},
 	"lisk": {
 		"minVersion": "1.1.0-rc.0",
 		"protocolVersion": "1.0"
-	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged",
```

Comment:
Same here

Reply:
it was `check:dependencies` that was running the snyk, but it doesn't exists anymore